feeadjuster/feeadjuster.py

```diff
@@ -138,6 +168,13 @@ def init(options: dict, configuration: dict, plugin: Plugin, **kwargs):
     "Note: it's also fuzzed by 1.5%",
     "string"
 )
+plugin.add_option(
+    "feeadjuster-ratio-base",
+    "50",
+    "Channel fees will be adjusted by the exponent of this."
+    "New fee = <default fee> * feeadjuster-ratio-base**(0.5 - <our liquidity ratio>)"
+    "string"
+)
```

Comment:
Re the option see the previous comment about how to imho cleanly upgrade the plugin :)